src/formBuilder/utils.js

```diff
@@ -512,6 +513,7 @@ export function generateElementPropsFromSchemas(parameters: {
   });
   // read dependent elements from dependencies
   if (schema.dependencies) {
+    const notUseReferenceDetails = false;
```

Comment:
Very confusing double-negative here.

Can we rather say:

```suggestion
    const useReferenceDetails = false;
```